test/ts/station_test.exs

```diff
@@ -93,29 +93,33 @@ defmodule StationTest do
 			congestion_low: 4, choose_fn: 1}
 
 		test_proc = self()
-		mock_send_to_stn = {fn(_) -> false end}
-		#Create mock station
-		{:ok, neighbour} = MockStation.start_link(mock_send_to_stn)
 
 		#Create NetworkConstructor Mock
 		MockRegister
-		|> expect(:check_active,
-			fn(_) -> send(test_proc, :query_received)
+		|> expect(:check_active, fn(_) -> send(test_proc, :query_received)
```

Comment:
It is a bad practice to have too many atoms in an application. This is because atoms are not garbage collected. Also by new atoms this refers to generating different atoms. So If I use :this_is_an_atom a million time no excess memory is used. 

From http://erlang.org/doc/efficiency_guide/advanced.html
By default, the maximum number of atoms is 1,048,576. This limit can be raised or lowered using the +t option.

The above entry refers to the number of distinct registered atoms in the atom table.

This link explains it pretty clearly. http://erlang.org/pipermail/erlang-questions/2007-November/030488.html

The atoms in the testing build will never impact the production build even if we by chance create 1M distinct atoms.